src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -292,8 +310,11 @@ export class ServiceSelectionScene extends SceneObjectBase<ServiceSelectionScene
     return `{${SERVICE_NAME}=\`${service}\`}${levelFilter}`;
   }
 
-  private getMetricExpression(service: string) {
-    return `sum by (${LEVEL_VARIABLE_VALUE}) (count_over_time({${SERVICE_NAME_EXPR}=\`${service}\`} | drop __error__ [$__auto]))`;
+  private getMetricExpression(service: string, serviceName: CustomConstantVariable) {
+    if (serviceName.state.value === AGGREGATED_SERVICE_NAME) {
+      return `sum by (${LEVEL_VARIABLE_VALUE}) (sum_over_time({${AGGREGATED_SERVICE_NAME}=\`${service}\`} | logfmt | drop __error__ | unwrap count [$__auto]))`;
+    }
+    return `sum by (${LEVEL_VARIABLE_VALUE}) (count_over_time({${SERVICE_NAME}=\`${service}\`} | drop __error__ [$__auto]))`;
```

Comment:
I don't _think_ we need the `drop __error__` here. If we did need it, it would mean that getting values from structured metadata could produce errors, which I don't think is possible, but will double check.

Reply:
so yeah, for a `count_over_time` using a structured metadata grouping I can't find a way it would produce an error

Reply:
I think we can remove `| drop __error__` from both expressions.

Reply:
yeah, good point, especially since we're doing the grouping by level, they'll effectively get dropped anyway

Reply:
for the `unwrap` we may want a `| __error__=""` instead, which will drop all lines that could not be unwrapped. However, that won't happen since we're creating this stream, so it might be a belt and suspenders situation

Reply:
Hmm, if something changes in the way we create the stream, and we do need to drop errors, that would mean we have a bug in the way we're creating the stream that we'd need to fix on the loki side. If we do drop errors then we still have a bug in loki but we would probably not catch it as quickly, so my argument would be to not drop or exclude errors in either query

Reply:
I agree